api.js

```diff
@@ -39,6 +74,12 @@ module.exports = {
         );
       },
       stop: (req, res) => {
+        const appRoot = req.app.locals.appRoot;
+        const webdashJson = require(`${appRoot}/webdash.json`);
+        if (~longRunningScripts.indexOf(body.script)) {
```

Comment:
why do longRunningScripts need to be stored in an object different than the one that already exists? which is `processes`